modules/terraform-guide-env-variables.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cloud_experts_tutorials/terraform/cloud-experts-terraform-customization.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="terraform-guide-env-variable_{context}"]
+= Using environmental variables
+
+You can use locally set variables to specify a value once and reuse that value later. 
+
+.Procedure
+
+. Set an environment variable by running the following command:
++
+[source,terminal]
+----
+$ export <1> = <2>
```

Comment:
```suggestion
$ export <1>=<2>
```
Spaces between the variable name and value are not supported in bash, so probably better to not include the spaces here